### PR TITLE
[11.0] [MIG] mis_builder_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of mis_buider_operating_unit to v11

### CODE REVIEW

File: mis_builder_operating_unit/security/mis_builder_security.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="0">
```
Comment (Nikul-Chaudhary) [inline mis_builder_operating_unit/security/mis_builder_security.xml]: @bjeficent no need `data`

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improved Code

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

File: mis_builder_operating_unit/model/mis_builder.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+from openerp import api, fields, models
+
+
+class MisReportInstance(models.Model):
+
+    _inherit = 'mis.report.instance'
+
+    operating_unit_ids = fields.Many2many('operating.unit',
+                                          string='Operating Unit',
+                                          required=False)
```
Comment (nikul-serpentcs) [inline mis_builder_operating_unit/model/mis_builder.py]: @bjeficent no need to `required=False`, It's already `False`

File: mis_builder_operating_unit/model/mis_builder.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+class MisReportInstancePeriod(models.Model):
+
+    _inherit = 'mis.report.instance.period'
+
+    operating_unit_ids = fields.Many2many('operating.unit',
+                                          string='Operating Unit',
+                                          required=False)
```
Comment (nikul-serpentcs) [inline mis_builder_operating_unit/model/mis_builder.py]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Review by enricostano — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

@nikul-serpentcs  can you update your review, please?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Review by sbidoul — APPROVED** [review]
LGTM

**Comment by TheCloneMaster** [comment]
@bjeficent do you have any idea of possible changes from v11 to v12?
I would like to help porting to v12...

**Comment by AaronHForgeFlow** [comment]
@TheCloneMaster Please see https://github.com/OCA/operating-unit/pull/211

**Pull request merged**